apps/reaction/package.json

```diff
@@ -71,6 +71,7 @@
     "@reactioncommerce/nodemailer": "5.0.5",
     "@reactioncommerce/random": "1.0.2",
     "@snyk/protect": "latest",
+    "bull": "4.10.1",
```

Comment:
I chose this one because their comment seems to imply that the typescript version was not as complete and well-tested. And since the last release was just 2 weeks ago it seems just as well maintained. "safe and battle tested" seemed like what I want

> If you want to start using the next major version of Bull written entirely in Typescript you are welcome to the new repo [here](https://github.com/taskforcesh/bullmq). Otherwise you are very welcome to still use Bull, which is a safe, battle tested codebase.